lib/ui/screens/editorial/editorial_screen.dart

```diff
@@ -76,29 +108,31 @@ class _WonderEditorialScreenState extends State<WonderEditorialScreen> {
           child: Stack(
             children: [
               /// Background
-              Positioned.fill(
-                child: ValueListenableBuilder(
-                  valueListenable: _scrollPos,
-                  builder: (_, value, __) {
-                    return Container(
-                      color: widget.data.type.bgColor.withOpacity(_scrollPos.value > 1000 ? 0 : 1),
-                    );
-                  },
+              ValueListenableBuilder<bool>(
+                valueListenable: _includeBackground,
+                builder: (context, include, child) => include ? child! : const SizedBox(),
+                child: Positioned.fill(
+                  child: Container(
```

Comment:
Since the only property that is being used here is color why not just use a ColoredBox instead?